examples/siro_sandbox/README.md

```diff
@@ -1,165 +1,136 @@
-# Sandbox Tool 
-
-![siro_sandbox_screenshot](https://user-images.githubusercontent.com/6557808/230213487-f4812c2f-ec7f-4d68-9bbe-0b65687f769b.png)
-
-# Overview
-This is a 3D interactive GUI app for testing various pieces of SIRo, e.g. rearrangement episode datasets, Fetch and Spot robots, humanoids (controllers, animation, skinning), trained agent policies, batch rendering and other visualization.
-
-# Known Issues
-* The tool is not very stable in the `SIRo` branch due to rapid iteration in various parts of the codebase. See Snapshots section below for best results.
-* The skinned humanoid doesn't render correctly. If you prefer, you can hide it with `--hide-humanoid-in-gui`.
-* When using HSSD scenes (see below), the app has bad runtime perf on older Macbooks (2021 is fine; 2019 is bad). See "Workaround for poor runtime perf on slower machines".
-
-# Snapshots with examples of running the tool
-See [SIRo Sandbox Snapshots Google Doc](https://docs.google.com/document/d/1cvKuXXE2cKchi-C_O7GGVFZ5x0QU7J9gHTIETzpVKJU/edit#). The tool is not very stable in the `SIRo` branch due to rapid iteration in various parts of the codebase. This doc describes well-tested sets of commits across our repos (Habitat-lab, Habitat-sim, fphab, floorplanner). This doc also gives example commands to run the tool.
-
-# Example commands
-### GUI-controlled humanoid and learned-policy-controlled Spot
-
-* To launch GUI-controlled humanoid and random-policy-controlled (initialized with random weights) Spot, in HSSD run:
-```
+# Sandbox Tool
```

Comment:
We should aim to (eventually) get rid of the "sandbox" term in favor of "HITL" everywhere.